python/tvm/driver/build_module.py

```diff
@@ -93,7 +93,7 @@ def schedule_to_module(
 def lower(
     inp: Union[schedule.Schedule, PrimFunc, IRModule],
     args: Optional[List[Union[Buffer, tensor.Tensor, Var]]] = None,
-    name: str = "main",
+    name: str = None,
```

Comment:
I think this might break a lot of things. "main" is used quite often as the default Function name in IRModules and I expect it'll show up in a lot of tests.

Reply:
Yeal, you are right. I finally fail to resolve all name errors related to unittests with a minor change. Is it acceptable if the PR only remove the name warning of #9602?

 If the default name is changed, currently broken testcases I have seen are:
- Test in microtvm require the function name `not` be `main` because of c code compilation.
- While a various set of tests use "main" as the expected IR function name.